web/webkit/src/test/scala/net/liftweb/http/provider/servlet/OfflineRequestSnapshotSpec.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2010-2011 WorldWide Conferencing, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.liftweb.http.provider.servlet
+
+import net.liftweb.http.provider._
+import net.liftweb.mockweb.WebSpec
+import org.mockito.Mockito._
+import org.specs2.mock.Mockito
+
+
+object OfflineRequestSnapshotSpec extends WebSpec with Mockito {
+
+  "A snapshot request should" in {
+    val mockHttpRequest = mock[HTTPRequest]
+    val mockProvider = mock[HTTPProvider]
```

Comment:
Setting the `context` value to `_` results in `unbound placeholder parameter`. I think that only works for class level `vars`.

Setting `context` to null doesn't feel right to me. I also thought about setting it to an instance of `net.liftweb.http.LiftFilter`, which implements the `HTTPProvider` trait, but then it would look like it has some purpose. So, considering these two scenarios, I think `mock` is probably appropriate here?